src/test/service/service.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * AT UI MOCKS - AT University Service API Test.
+ * Copyright 2020 AgileThought, Inc.
+ *
+ * Integration Test for at-university-api endpoint.
+ *
+ * @author @humberto-gonzalez
+ * @version 1.0
+ */
+
+// Constants
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const expect = require("chai").expect;
+chai.use(chaiHttp);
+
+// AT University Service API
+const tutorialServiceAPI =
+  "https://at-university-api.herokuapp.com/api" + "/v1/course";
+const tutorialServiceAPI_400 =
+  "https://at-university-api.herokuapp.com/api" + "/v1/courses";
+
+describe("TEST: at-univeristy.service", () => {
+  it("GET: Should Get All Courses - 200", async () => {
+    chai
+      .request(tutorialServiceAPI)
+      .get("/")
+      .end(function (err, res) {
+        //console.log(res.body);
+
+        // Response Status
+        expect(res).to.have.status(200);
+
+        done();
+      });
+  });
+});
+
+it("Should Fail Get All Courses - 400", () => {
+  chai
+    .request(tutorialServiceAPI_400)
+    .get("/")
+    .end(function (err, res) {
+      console.log(res.body);
+
+      // Response Status
+      expect(res).to.have.status(404);
+
+      // Response message
```

Comment:
try to cover more scenarios expect(res.body).to.have.property('status').equals(404);